### PR TITLE
Replace ember-cli-eslint with eslint

### PR DESCRIPTION
**Pull request by SergeAstapov**
Part of addon modernization efforts

### CODE REVIEW

File: test-projects/01-basic-app/tests/acceptance/identity-manager-test.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
-import { test } from 'qunit';
-import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+import { module, test } from "qunit";
+import { setupApplicationTest } from "ember-qunit";
+import { setupMirage } from "ember-cli-mirage/test-support";
 
-moduleForAcceptance('Acceptance | Identity manager');
+module('Acceptance | Identity manager', function(hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks)
```
Comment (cah-briangantzler) [inline test-projects/01-basic-app/tests/acceptance/identity-manager-test.js:7]: How was this test working before if they didnt call setupMirage?

**Comment by cah-briangantzler** [comment]
It cant find the lint command since this PR is dependent on the PR that adds lint. I have merged that. Please rebase this on master and submit again.

File: test-projects/01-basic-app/tests/acceptance/identity-manager-test.js (hunk 1 above)
Comment (SergeAstapov) [reply test-projects/01-basic-app/tests/acceptance/identity-manager-test.js:7]: @cah-briangantzler I'm not sure tbh. I think there was some automatic mirage start with old acceptance tests setup but I didn't bother to find out

**Comment by SergeAstapov** [comment]
@cah-briangantzler rebased, now some tests fail - I'll fix those.

**Comment by SergeAstapov** [comment]
@cah-briangantzler CI is happy now (except of Ember v4 scenarios which I'm fixing in other PRs)

**Pull request merged**